tmuxinator.gemspec

```diff
@@ -42,5 +42,14 @@ Gem::Specification.new do |s|
   s.add_dependency "erubis", "~> 2.6"
 
   s.add_development_dependency "bundler", "~> 1.3"
+  s.add_development_dependency "rake", "~> 10.4"
+  s.add_development_dependency "rspec", "~> 3.3"
+  s.add_development_dependency "simplecov", "~> 0.11.0"
+  s.add_development_dependency "coveralls", "~> 0.7"
+  s.add_development_dependency "awesome_print", "~> 1.2"
+  s.add_development_dependency "pry", "~> 0.10"
+  s.add_development_dependency "factory_girl", "~> 4.5"
+  s.add_development_dependency "activesupport", "< 5.0.0"  # Please see issue #432
```

Comment:
I included the activesupport gem because it was in the original Gemfile. I can remove it from the gemspec if it is not required.